### PR TITLE
Chore Fixes

### PR DESCRIPTION
**Pull request by Drapegnik**

### CODE REVIEW

File: src/db/scripts/initAll.js [hunk 1]
```diff
@@ -2,17 +2,16 @@
 
 // The script updates databases with data from Fibery.io.
 // The script is executed as follows:
-//   npm run init-db -- path-to-secret-file
+//   npm run init-db-secret
```
Comment (UladBohdan) [inline src/db/scripts/initAll.js]: are you updating a script name here?

File: src/api/files.js [hunk 2]
```diff
@@ -10,8 +10,12 @@ import { imagesDir } from 'utils/args';
...
 const { host, token } = config.services.fibery;
 
-const filesProxy = async ({ params: { secret }, query: { w = '', f = 'png' } }, res, next) => {
-  const filepath = `${imagesDir}/${secret}_${w}.${f}`;
+const filesProxy = async (
+  { params: { secret }, query: { w = '', h = '', f = 'png' } },
+  res,
+  next
+) => {
+  const filepath = `${imagesDir}/${secret}_${w}x${h}.${f}`;
```
Comment (UladBohdan) [inline src/api/files.js]: what if `h` is passed but `w` is not? `x` still will be a marker on how the image was resized?

Comment (Drapegnik) [reply src/api/files.js]: yep

File: src/db/scripts/initAll.js (hunk 1 above)
Comment (Drapegnik) [reply src/db/scripts/initAll.js]: some time ago

File: src/db/scripts/initAll.js [hunk 3]
```diff
@@ -2,17 +2,16 @@
...
 //
 // Examples:
 // * to populate local db:
 //   npm run init-db
```
Comment (UladBohdan) [inline src/db/scripts/initAll.js]: `-secret` is missing here and below

File: src/db/scripts/initAll.js [hunk 4]
```diff
@@ -81,27 +80,31 @@ export const initMainPageState = async metadataTestingUser => {
   const tags = await Tag.find().exec();
   const tagsBySlug = keyBy(tags, 'slug');
   const articlesByTag = getArticlesByTag({ articles, tags });
-  const { personalities = [], locations = [] } = getTagsByTopic({ tags, topics });
-
-  // TODO: filter out personalities with 0 articles (diary authors)
```
Comment (UladBohdan) [inline src/db/scripts/initAll.js]: now we're doing it both on Backend and Frontend ?

Comment (Drapegnik) [reply src/db/scripts/initAll.js]: in different places

**Pull request merged**